resources/config.rb

```diff
@@ -22,38 +22,60 @@
 property :root_group, String,
          default: lazy { default_apache_root_group },
          description: 'Group that the root user on the box runs as. Defaults to platform specific locations, see libraries/helpers.rb'
+
 property :access_file_name, String,
          default: '.htaccess',
          description: 'String: Access filename'
```

Comment:
This is going to cause failures on all but the newest Chef 13s

Reply:
How's that going to cause an error?

Reply:
description wasn't a method on property back in the day

Reply:
Ah jeez, so this will break for anyone who isn't running the latest 13? 😢

Reply:
I'd have to see when we added it to 13, but I think it was 13.11.